src/pages/csharp/operators/logical/index.md

```diff
@@ -0,0 +1,40 @@
+# Logical Operators
```

Comment:
Add before first line in this code:
```
---
title: Logical Operators
---
```

Reply:
Thanks Daria, first time contributing so I was bound to get something wrong! Edited it in.